.github/pull_request_template.md

```diff
@@ -14,7 +14,8 @@
 * [ ] All new Python files are placed inside an existing directory.
 * [ ] All filenames are in all lowercase characters with no spaces or dashes.
 * [ ] All functions and variable names follow Python naming conventions.
-* [ ] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
+* [ ] All function parameters and return values are annotated with Python [mypy](https://docs.python.org/3/library/typing.html).
 * [ ] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
 * [ ] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
+* [ ] Modified/Added code follows [black](https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html) and [flake8](https://flake8.pycqa.org/en/latest/) code style.
```

Comment:
We have tests for this so we do not need to ask the contributor if they have done this.